.idea/libraries/extensions.xml

```diff
@@ -8,7 +8,9 @@
       <root url="file://$PROJECT_DIR$/extensions/modules/lib" />
     </CLASSES>
     <JAVADOC />
-    <SOURCES />
+    <SOURCES>
+      <root url="jar://$USER_HOME$/.ideaLibSources/lucene-analyzers-common-4.10.4-sources.jar!/" />
```

Comment:
not sure if this should be in the PR

Reply:
@dizzzz It won't hurt. But I can remove it if you prefer?